docker/builder/tools.Dockerfile

```diff
@@ -31,6 +31,10 @@ COPY --link --from=tools-builder /aptos/dist/aptos /usr/local/bin/aptos
 COPY --link --from=tools-builder /aptos/dist/aptos-openapi-spec-generator /usr/local/bin/aptos-openapi-spec-generator
 COPY --link --from=tools-builder /aptos/dist/aptos-fn-check-client /usr/local/bin/aptos-fn-check-client
 COPY --link --from=tools-builder /aptos/dist/aptos-transaction-emitter /usr/local/bin/aptos-transaction-emitter
+COPY --link --from=tools-builder /aptos/dist/aptos-api-tester /usr/local/bin/aptos-api-tester
+
+# Copy the example module to publish for api-tester
+COPY --link --from=tools-builder /aptos/aptos-move /aptos-move
```

Comment:
Kind of a large folder, what's the exact thing we're copying over?

Reply:
We are compiling /aptos-move/move-examples/hello_blockchain", however copying the lower level folder throws dependency errors